src/libsync/propagatecommonzsync.cpp

```diff
@@ -81,7 +82,13 @@ void ZsyncSeedJob::run()
     zsyncControlFile.close();
     rewind(f);
 
-    zsync_unique_ptr<struct zsync_state> zs(zsync_begin(f), [](struct zsync_state* zs) {
+
+    QTemporaryDir tmpDir;
+    tmpDir.setAutoRemove(false);
+    std::string tmp_dir = tmpDir.path().toStdString();
+    const char * dname = tmp_dir.c_str();
```

Comment:
Does zsync create multiple files? If it's just one, could we make it use the temporary file name we already create?

Reply:
Added: What you have right now would be ok for the initial version

Reply:
Already fixed this, see ahmedammar/client@e31b69d.